### PR TITLE
Added kubemonkey to services phase

### PR DESCRIPTION
**Pull request by sandipndev**

### CODE REVIEW

File: modules/services/kubemonkey.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+variable "dry_run" { default = true }
```
Comment (bodymindarts) [inline modules/services/kubemonkey.tf:1]: Variables belong in `modules/services/variables`

File: modules/services/variables.tf [hunk 2]
```diff
@@ -1,7 +1,7 @@
 variable "name_prefix" {}
 variable "cluster_endpoint" {}
 variable "cluster_ca_cert" {}
-variable "honeycomb_api_key" {}
+variable "secrets" {}
```
Comment (bodymindarts) [inline modules/services/variables.tf:4]: cannot merge this before changes in galoy-org-infra

File: modules/services/kubemonkey.tf (hunk 1 above)
Comment (sandipndev) [reply modules/services/kubemonkey.tf]: Done

File: modules/services/kubemonkey-values.yml.tmpl [hunk 3]
```diff
@@ -0,0 +1,16 @@
+config:
+  dryRun: false
+  runHour: 0
+  startHour: 3
+  endHour: 5
```
Comment (sandipndev) [inline modules/services/kubemonkey-values.yml.tmpl:5]: @bodymindarts I also decided to remove the variables for these parameters (adding them was premature) and stick to these defaults.

**Pull request merged**